src/Mobs/PathFinder.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+class cPathFinder;
+class cPath;
+class cChunk;
+
+class cPathFinder
+{
+public:
+	cPathFinder(double a_BoundingBoxWidth, double a_BoundingBoxHeight);
+	Vector3d GetWayPoint(cChunk & a_Chunk, const Vector3d & a_CurrentLocation, Vector3d & a_Destination);
```

Comment:
a_Destination is not a const because PathFinder may decide that it's a bad destination and pick a close one. This is probably not the best way to do it and should be changed. PathFinder should probably signal that it only found a nearby path.